packages/web/src/common/store/pages/collection/lineups/sagas.js

```diff
@@ -28,70 +23,12 @@ function* getCollectionTracks() {
     collection = yield select(getSmartCollection, {
       variant: smartCollectionVariant
     })
+    return collection.tracks
   } else {
-    collection = yield call(waitForValue, getCollection)
+    const collectionId = yield select(getCollectionId)
+    const collection = yield select(getCollection, { id: collectionId })
+    return collection.tracks
   }
-
-  const tracks = collection.playlist_contents.track_ids
-
-  const trackIds = tracks.map((t) => t.track)
-  // TODO: Conform all timestamps to be of the same format so we don't have to do any special work here.
-  // Some playlists & albums may not have a metadata time, so use the time if not.
-  // If for whatever reason, the metadata time is off, prefer the release date
-  const times = tracks.map((t) =>
-    Math.max(t.metadata_time || t.time, moment(collection.release_date).unix())
-  )
-
-  // Reconcile fetching this playlist with the queue.
-  // Search the queue for its currently playing uids. If any are sourced
-  // from `this` collection, use their uids rather than allowing
-  // the lineup to generate fresh ones.
-  // This allows the user to navigate back and forth between playlists and other
-  // pages without losing their currently playing position in the playlist.
```

Comment:
we're not adding support for this yet, right?
add a todo and/or ticket unless you think it's implied in lineup work